snapcraft.yaml

```diff
@@ -496,6 +496,10 @@ parts:
       - libxi-dev
       - libxkbfile-dev
       - libxml2-utils
+    stage:
+      - -usr/bin/wayland-scanner
+      - -usr/lib/x86_64-linux-gnu/libwayland-client.so.0.20.0
```

Comment:
It would be better to do this without the arch specific path, replace x86_64-linux-gnu with "*"

Reply:
Good point.